mage_integrations/mage_integrations/sources/salesforce/README.md

```diff
@@ -11,13 +11,35 @@ You must enter the following settings when configuring this source:
 | Key | Description | Sample value
 | --- | --- | --- |
 | `api_type` | The `api_type` is used to switch the behavior of the tap between using Salesforce's "REST" and "BULK" APIs. When new fields are discovered in Salesforce objects, the `select_fields_by_default` key describes whether or not the tap will select those fields by default. | `REST`, `BULK` |
-| `client_id` | OAuth Salesforce App secrets. | `ABC1...` |
-| `client_secret` | OAuth Salesforce App secrets. | `ABC1...` |
-| `is_sandbox` | If `true`, then the sandbox account will be used to load dat from. | `true`, `false` |
-| `lookback_window` | The `lookback_window` (in seconds) subtracts the desired amount of seconds from the bookmark to sync past data. Recommended value: 10 seconds. | `10` |
-| `refresh_token` | The `refresh_token` is a secret created during the OAuth flow | `ABC1...` |
+| `credentials` | Dict containing simple-salesforce login data, can be either `OAUTH` or `PASSWORD`, please check below for detailed info.
```

Comment:
Let's make the config backward compatible.

Reply:
Added in 8cc2a0e2931ffc6e60ebd0611b0fa1521525ccda